### PR TITLE
Use Appsignal.Utils.compile_env in Appsignal.Phoenix

### PR DESCRIPTION
**Pull request by michallepicki**
to avoid warnings on Elixir 1.14+

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -55,7 +55,7 @@ defmodule Appsignal.Phoenix.MixProject do
       end
 
     [
-      {:appsignal_plug, ">= 2.0.8 and < 3.0.0"},
+      {:appsignal_plug, ">= 2.0.9 and < 3.0.0"},
```
Comment (michallepicki) [inline mix.exs:58]: The lower bound bump is for the transitive dependency `appsignal` version which provides `Appsignal.Utils`.
(btw maybe it should be a direct dependency instead)

Comment (jeffkreeftmeijer) [reply mix.exs:58]: Thanks! This makes sure the dependency for AppSignal is above 2.0.8 (specifically 2.0.10), which adds `Appsignal.Utils.compile_env`.

**Comment by jeffkreeftmeijer** [comment]
Thanks @michallepicki, good catch! I’ll get this one released, and will update your on that here.

**Comment by jeffkreeftmeijer** [comment]
Update: I’ve just released version 2.0.14 of the appsignal_phoenix library, which includes this patch.

**Pull request merged**